nengo_gui/components/pointer.py

```diff
@@ -53,11 +69,12 @@ def gather_data(self, t, x):
         msg = '%g %s' % (t, text)
         self.data.append(msg)
         if self.override_target is None:
-            return self.vocab_in.parse('0').v
+            return self.default_override_val
         else:
             v = (self.override_target.v - x) * 3
-            if self.vocab_in is not self.vocab_out:
-                v = np.dot(self.vocab_out.transform_to(self.vocab_in), v)
+            if self.loop_in:
+                if self.vocab_in is not self.vocab_out:
+                    v = np.dot(self.vocab_out.transform_to(self.vocab_in), v)
```

Comment:
Although before I suggest removing it from the whitelist, how well did it work when you tried it?  What sort of things did you try?